tools/setup.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Use this script to create a wheel with OpenVINO™ Python* tools:
+
+$ python setup.py sdist bdist_wheel
+"""
```

Comment:
add notes that it works only with openvino pkg or add it as dependency

Reply:
Added to the install_requires list